src/Avalonia.Controls/TransitioningContentControl.cs

```diff
@@ -12,8 +13,11 @@ namespace Avalonia.Controls;
 /// Displays <see cref="ContentControl.Content"/> according to an <see cref="IDataTemplate"/>,
 /// using a <see cref="PageTransition"/> to move between the old and new content. 
 /// </summary>
+[PseudoClasses(_PcReversed)]
```

Comment:
Is a pseudoclass necessary for this feature? What use-cases did you envision needing a pseudoclass for? They're usually only created for cases where styles will commonly need to select on the property.

Reply:
Yeah, I questioned that myself.  And at the time I submitted, I hadn't discovered that you can use properties directly in the selector.  I don't think it's needed, I can remove it.